assets/vue/components/social/SocialWallPost.vue

```diff
@@ -108,17 +103,33 @@ const store = useStore();
 
 const { relativeDatetime } = useFormatDate()
 
-const attachment = null;//props.post.attachments.length ? props.post.attachments[0] : null;
 let comments = reactive([]);
+const attachments = ref([]);
+const currentUser = store.getters['security/getUser'];
+const isOwner = computed(() => currentUser['@id'] === props.post.sender['@id'])
 
-const containsImage = false; //attachment && attachment.resourceNode.resourceFile.mimeType.includes('image/');
-const containsVideo = false; //attachment && attachment.resourceNode.resourceFile.mimeType.includes('video/');
+onMounted(async () => {
+  loadComments();
 
-const currentUser = store.getters['security/getUser'];
+  await loadAttachments();
+});
 
-const isOwner = computed(() => currentUser['@id'] === props.post.sender['@id'])
+const computedAttachments = computed(() => {
+  return attachments.value;
+});
 
-onMounted(loadComments);
+async function loadAttachments() {
+  try {
+    let idUrl = props.post["@id"];
+    let parts = idUrl.split('/');
+    let socialPostId = parts[parts.length - 1];
+
+    const response = await axios.get(`${ENTRYPOINT}social_posts/${socialPostId}/attachments`);
+    attachments.value = response.data;
+  } catch (error) {
+    console.error("There was an error loading the attachments!", error);
+  }
+}
```

Comment:
Si es un subresource, entonces se debería reutilizar el mismo IRI (o URL) del SocialPost.

```suggestion
async function loadAttachments() {
  try {
    const postIri = props.post["@id"]

    const response = await axios.get(`${postIri}/attachments`)
    attachments.value = response.data
  } catch (error) {
    console.error("There was an error loading the attachments!", error)
  }
}
```